src/SonarQube.Client.Tests/Requests/Api/V7_20/GetIssuesRequestTests.cs

```diff
@@ -462,7 +462,7 @@ public async Task InvokeAsync_ComponentKeySpecified_ComponentsAreIncludedInQuery
             _ = await testSubject.InvokeAsync(httpClient, CancellationToken.None);
 
             var actualQueryString = GetSingleActualQueryString(handlerMock);
-            actualQueryString.Contains("components=project1").Should().BeTrue();
+            actualQueryString.Contains("components=project1").Should().BeFalse();
```

Comment:
I would reduce the risk for miss-asserting like this (even removing the "s")
```suggestion
            actualQueryString.Contains("component").Should().BeFalse();
```